src/test/java/org/apache/ibatis/executor/loader/ResultLoaderMapTest.java

```diff
@@ -0,0 +1,49 @@
+/**
+ *    Copyright 2009-2019 the original author or authors.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package org.apache.ibatis.executor.loader;
+
+import java.sql.SQLException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ResultLoaderMapTest {
+
+  @Test
+  void hasLoaderFalse() {
+    assertFalse(new ResultLoaderMap().hasLoader("==="));
+  }
+
+  @Test
+  void loadAllNullPointerException() throws NullPointerException, SQLException {
+    new ResultLoaderMap().loadAll();
+  }
+
+  @Test
+  void loadInputNotNullOutputFalse() throws SQLException {
+    assertFalse(new ResultLoaderMap().load("]]]"));
+  }
+
+  @Test
+  void removeInputNotNullOutputVoid() {
+    new ResultLoaderMap().remove("]]]]]]]");
+  }
+
+  @Test
+  void sizeOutputZero() {
```

Comment:
Would it be possible to write a test here that constructs a ResultLoaderMap() with a size of > 0?

Reply:
Could be done:
1. instantiate a new `ResultLoaderMap resultLoaderMap`
2. instantiate a `MetaObject` and a `ResultLoader`
3. call `resultLoaderMap.addLoader("some string", metaObject, resultLoader)`

Reply:
On closer inspection (with @louismillsdiffblue ) this is probably very difficult.